### DISCUSSION TITLE
Accomplishments that we are proud of

### DISCUSSION THREAD

**Discussion by xavier506** [eoscostarica/gGoods#199] [General]
- Researching all the different NFT standards in EOSIO and other blockchains to determine the right features for gGoods.
- Integrating payment gateways like Google Pay and Paypal for the purchase of NFTs
- Integrating Augmented Reality and progressive web applications into the gGoods in-app usage “selfie cam”.
- Making the purchase of an NFT as easy as donating to an NGO.
- Using blockchain technology to encourage anyone to donate their time, creativity and uniqueness to a cause they support. Not just money.

**Comment by xavier506** [comment]
- @kriskoin  wrote unit test for smart contracts!

**Comment by kriskoin** [comment]
i  proud of how speed up the deployment process on bare metal environment,
